mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -22,8 +22,14 @@ function MentoringAssessmentView(runtime, element, mentoring) {
         messagesDOM.empty().hide();
     }
 
+    function no_more_attempts() {
+        var attempts_data = $('.attempts', element).data();
+        return attempts_data.num_attempts >= attempts_data.max_attempts;
+    }
+
     function renderGrade() {
         var data = $('.grade', element).data();
+        data['enable_extended'] = (no_more_attempts() && data['extended_feedback']);
```

Comment:
OCD fueled nitpick: We use the dot notation for accessing properties elswhere in the JS code, so I'd prefer if this was `data.enable_extended` and `data.extended_feedback`.